contracts/OpenSTUtility.sol

```diff
@@ -107,7 +107,7 @@ contract OpenSTUtility is Hasher, OpsManaged {
 	uint256 public chainIdUtility;
 	address public registrar;
 	/// registered branded tokens 
-	mapping(bytes32 /* uuid */ => RegisteredToken) public registeredTokens;
+	mapping(bytes32 /* uuid */ => RegisteredToken) registeredTokens;
```

Comment:
because it s a struct, and so a public getter cannot be implemented by the compiler (it cannot return a struct, correct?); in exchange we have `registeredTokenProperties`

Reply:
But then I think that should mean we are seeing compiler errors on `develop`. Also, when I called the public function just now in what's on the `develop` branch, it returns two (sensible looking) addresses.

Reply:
so then structs can be auto-decomposed by the compiler; there does not need to be both though.  Then we should remove likewise functions on OpenSTValue.sol as well; I'll open an issue for it instead, that updates the integration tests that use these getters.